### ISSUE TITLE
[Accessibility]: Code Insights Creation UI - Preview loading does not announce what is loading

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41671]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33494

### Problem description

![PixelSnap 2022-09-14 at 14 32 31](https://user-images.githubusercontent.com/1855233/190245846-b3006371-bb02-4fbf-a73b-a828e36e3978.png)

When the live preview button is clicked, or any time any data is updated that causes this chart to refresh, the screen reader announces "loading."

### Expected behavior

Should announce "what" is loading. Something like "Loading live preview chart."

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
